src/text.css

```diff
@@ -86,32 +86,32 @@ h6 {
 
 h1,
 .h1 {
-  font-size: var(--headings-font-size-h1);
+  font-size: calc(var(--headings-font-size-h1) * var(--media-h));
```

Comment:
`media` here does not really follow the pattern of using `m` to describe media in class names. comparing to `.m-md-medium` and others

Reply:
my choice would be to expand the class name from `.m...` to `.media...` and just make it more verbose